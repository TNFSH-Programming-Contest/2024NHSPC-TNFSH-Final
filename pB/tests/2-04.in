100 12
10 3 8 10 3 10 6 2 3 10 2 5 6 6 6 10 4 1 9 1 7 2 2 2 6 7 1 5 2 9 7 2 10 10 2 4 3 3 6 9 8 7 3 4 9 8 8 3 1 6 8 9 6 9 4 8 10 9 5 8 6 10 10 1 9 4 3 2 7 4 7 3 6 8 9 3 3 5 2 6 8 2 8 5 1 8 5 4 7 2 3 9 7 7 10 4 3 1 2 8
1488920 63 12607 77818 46 5 287 26 154972 13651085 93510 49317574
0 
3 10 2 7
41
0 
7 12 11 9 6 5 3 1
42
0 
2 1 8
3
0 
8 1 6 12 7 3 11 8 2
46
0 
3 3 1 11
72
0 
1 3
41
0 
10 6 3 8 12 4 11 7 5 9 2
51
0 
9 9 6 2 1 4 3 8 5 7
22
0 
5 4 5 10 11 12
14
0 
7 3 1 8 10 4 7 11
62
0 
9 10 6 7 9 11 3 2 12 8
35
0 
10 8 1 2 7 4 9 3 12 10 6
33
0 
8 2 12 9 10 8 1 4 7
79
0 
8 4 1 5 10 3 7 11 12
45
0 
10 4 11 9 8 3 1 6 12 10 2
61
0 
9 4 11 1 3 2 8 9 12 10
89
0 
8 3 11 5 7 12 9 4 10
66
0 
5 9 2 7 10 5
95
0 
2 2 8
57
0 
6 8 11 3 1 6 9
26
0 
8 9 12 4 8 7 10 11 2
35
0 
7 10 11 12 6 7 9 4
27
0 
2 8 2
12
0 
6 10 11 4 12 8 5
93
0 
9 4 1 5 11 6 2 8 9 7
78
0 
0 
40
0 
1 6
83
0 
4 4 6 12 11
32
0 
1 5
4
0 
1 1
70
0 
4 5 8 7 3
45
0 
2 3 9
62
0 
4 12 3 11 6
34
0 
6 7 4 1 6 2 3
65
0 
6 4 3 10 1 8 7
28
0 
4 3 8 12 6
46
0 
4 11 1 5 12
1
0 
2 8 6
11
0 
10 11 6 9 12 8 2 3 10 5 7
84
0 
4 11 7 2 1
29
0 
3 10 3 8
13
0 
8 10 1 3 7 12 2 6 11
51
0 
10 12 1 2 11 9 6 8 5 10 4
80
0 
0 
95
0 
4 5 10 11 2
91
0 
4 2 7 12 3
98
0 
4 3 10 12 1
90
0 
0 
4
0 
10 10 3 9 7 11 12 1 5 2 6
71
0 
2 11 5
13
0 
4 12 5 3 2
28
0 
2 6 12
24
0 
0 
76
0 
5 12 11 10 4 8
10
0 
4 10 3 1 12
51
0 
10 6 2 10 12 11 4 9 7 1 3
52
0 
4 8 2 1 5
81
0 
8 12 4 8 11 7 1 3 2
65
0 
2 3 5
97
0 
6 10 1 5 9 11 2
71
0 
5 1 12 9 8 7
50
0 
8 9 8 10 3 2 5 7 11
40
0 
9 8 11 9 7 3 2 6 5 1
62
0 
0 
34
0 
2 10 7
92
0 
8 10 3 6 1 4 5 11 9
75
0 
1 12
30
0 
1 10
36
0 
10 2 9 10 12 11 1 6 4 5 3
98
0 
9 10 2 6 3 9 12 5 4 7
75
0 
2 1 4
26
0 
4 12 8 3 1
13
0 
6 2 12 5 8 1 11
29
0 
4 7 4 1 3
67
0 
5 3 7 11 6 8
91
0 
8 2 10 5 12 8 6 1 4
86
0 
7 11 2 1 10 6 5 4
86
0 
1 11
18
0 
1 12
97
0 
10 11 12 5 6 2 3 4 1 9 8
28
0 
1 10
84
0 
0 
76
0 
4 1 6 8 12
17
0 
2 10 1
78
0 
3 3 11 12
18
0 
5 12 10 5 4 6
54
0 
7 4 5 9 3 12 10 1
73
0 
1 4
91
0 
8 6 2 12 11 8 10 9 7
89
0 
10 11 12 8 10 7 1 9 6 4 3
59
0 
0 
76
0 
10 10 2 7 11 8 6 3 5 9 1
59
0 
10 8 4 9 5 3 6 10 2 7 11
92
0 
2 6 1
81
0 
2 10 1
58
0 
4 5 10 8 6
60
0 
1 10
67
0 
10 12 1 7 3 4 9 2 8 6 10
49
0 
7 7 1 11 9 3 8 5
33
0 
4 9 4 12 2
19
0 
0 
91
0 
3 11 2 7
20
0 
0 
26
0 
5 11 12 10 4 6
92
0 
1 7
57
0 
9 7 12 11 9 1 3 2 6 5
80
0 
3 2 5 3
48
0 
2 8 1
54
0 
1 1
43
0 
2 2 3
54
0 
6 3 11 8 1 6 12
52
0 
8 10 2 6 3 1 9 8 4
13
0 
6 4 12 5 11 8 7
35
0 
1 3
78
0 
4 11 7 3 8
57
0 
7 5 4 7 8 1 3 9
68
0 
5 10 9 12 5 11
41
0 
7 7 5 3 12 9 8 1
24
0 
2 12 11
49
0 
9 4 9 6 2 3 12 7 11 10
22
0 
9 8 5 6 2 4 3 11 10 12
41
0 
2 9 12
7
0 
2 9 2
20
0 
4 3 9 6 1
38
0 
9 2 4 7 10 12 6 1 3 11
64
0 
2 6 8
42
0 
0 
10
0 
8 3 1 7 5 10 12 9 11
18
0 
5 1 7 3 4 11
35
0 
1 8
69
0 
1 5
54
0 
9 8 3 1 12 7 9 4 2 10
41
0 
9 11 4 2 9 8 7 6 1 5
55
0 
2 11 4
90
0 
8 3 5 8 10 6 2 1 9
99
0 
4 2 4 3 1
56
0 
5 1 8 4 9 10
93
0 
10 2 5 6 4 9 8 7 12 11 1
7
0 
6 6 12 4 8 5 9
79
0 
4 7 3 4 1
20
0 
3 2 12 8
100
0 
7 12 10 9 11 8 7 4
13
0 
3 10 4 6
80
0 
10 11 9 4 2 6 8 7 1 10 3
46
0 
7 9 8 12 5 11 10 4
99
0 
8 7 2 9 4 3 11 1 6
69
0 
2 6 12
53
0 
4 3 2 4 5
53
0 
2 8 2
28
0 
7 9 2 6 10 11 8 4
15
0 
4 3 8 4 11
0
0 
0 
72
0 
8 10 12 5 11 2 6 7 8
91
0 
0 
59
0 
7 5 12 9 6 8 10 1
6
0 
7 7 4 11 10 9 6 1
99
0 
3 5 1 3
37
0 
0 
29
0 
3 9 6 1
7
0 
5 7 11 1 4 3
71
0 
1 8
25
0 
2 5 12
93
0 
3 1 9 10
89
0 
8 12 8 2 7 11 6 1 10
40
0 
1 8
9
0 
3 11 4 7
36
0 
6 2 4 9 6 7 5
13
0 
7 11 7 6 3 8 5 9
90
0 
0 
74
0 
4 6 11 5 8
93
0 
8 9 3 4 2 8 6 5 10
8
0 
9 10 12 2 11 9 5 6 7 4
4
0 
2 8 6
83
0 
5 7 8 1 12 2
53
0 
7 7 12 6 9 11 1 10
16
0 
1 5
13
0 
1 3
90
0 
1 8
63
0 
5 11 2 7 1 3
40
0 
1 7
9
0 
8 5 3 11 6 2 4 10 8
32
0 
6 9 2 7 4 6 8
49
0 
6 6 3 5 12 8 2
38
0 
10 6 4 8 7 12 2 3 9 1 5
95
0 
10 11 4 7 2 1 6 8 3 5 12
78
0 
3 3 9 4
44
0 
7 7 9 5 4 12 6 1
49
0 
9 12 11 7 2 8 6 1 4 3
67
0 
1 5
96
0 
3 1 12 10
83
0 
9 7 3 5 11 4 6 9 12 8
93
0 
4 9 10 6 5
26
0 
8 10 6 9 11 5 12 3 4
88
0 
10 12 10 1 9 4 2 8 6 3 5
37
0 
10 12 10 1 8 11 2 7 5 4 9
81
0 
4 11 2 6 10
69
0 
10 9 7 2 1 11 10 12 3 8 5
16
0 
5 1 6 11 8 9
60
0 
8 1 9 11 10 2 8 7 5
78
0 
5 1 6 7 9 11
65
0 
0 
12
0 
7 7 1 6 8 9 4 11
15
0 
0 
68
0 
0 
39
0 
9 8 12 9 10 6 1 11 4 3
89
0 
4 10 2 11 5
76
0 
5 12 2 8 3 4
43
0 
9 7 8 3 4 11 5 1 10 9
29
0 
9 6 11 3 4 12 2 5 8 9
52
0 
9 4 2 6 12 11 8 1 10 3
63
0 
9 3 7 4 5 12 6 11 1 9
67
0 
5 7 3 2 11 9
39
0 
10 4 3 2 8 1 11 9 6 10 12
42
0 
10 3 10 5 7 2 4 12 1 9 11
98
0 
4 10 7 11 9
34
0 
3 2 6 12
31
0 
6 5 1 9 12 7 4
75
0 
6 3 8 6 9 2 10
59
0 
8 4 3 12 6 1 11 8 7
40
0 
8 7 1 11 3 10 8 4 2
39
0 
3 9 2 6
64
0 
2 9 7
16
0 
6 8 11 7 3 5 9
96
0 
2 6 12
31
0 
8 10 1 9 2 6 4 7 12
50
0 
10 6 3 9 8 4 11 7 5 10 12
96
0 
3 8 12 9
23
0 
5 11 9 7 6 4
3
0 
1 2
70
0 
1 9
91
0 
2 7 3
70
0 
7 7 6 9 10 3 8 1
24
0 
6 7 4 6 1 5 3
44
0 
10 4 5 10 12 11 2 7 8 9 1
18
0 
6 2 12 1 3 4 7
32
0 
9 5 1 11 8 2 4 6 7 3
33
0 
5 4 12 9 5 3
52
0 
3 6 7 4
71
0 
3 1 11 9
57
0 
1 5
77
0 
4 2 12 3 1
38
0 
8 11 1 9 4 7 6 3 10
71
0 
4 5 7 10 12
98
0 
3 8 5 1
95
0 
3 6 5 11
7
0 
9 1 3 4 7 9 11 10 12 8
74
0 
5 9 2 12 3 1
35
0 
2 2 3
58
0 
10 10 1 6 11 8 12 7 4 2 3
55
0 
1 4
98
0 
9 5 7 4 3 8 10 1 11 2
96
0 
9 10 9 3 6 12 2 4 11 5
14
0 
2 6 9
27
0 
5 3 7 12 4 5
63
0 
3 3 10 6
95
0 
10 7 4 8 9 11 1 6 12 2 10
32
0 
6 5 2 10 1 8 7
1
0 
0 
15
0 
4 1 2 6 5
53
0 
1 6
91
0 
5 7 9 11 4 8
59
0 
7 3 11 7 8 10 12 4
14
0 
0 
95
0 
8 6 10 4 11 5 3 9 7
85
0 
2 8 5
64
0 
1 2
63
0 
4 5 11 7 1
9
0 
4 11 8 7 4
36
0 
10 3 11 2 1 6 4 9 12 8 5
34
0 
4 4 7 3 8
46
0 
4 9 6 8 11
65
0 
7 8 12 1 2 3 6 9
4
0 
7 8 12 11 7 1 5 2
2
0 
10 1 6 10 12 9 11 3 5 2 7
91
0 
8 8 4 12 2 11 1 7 5
26
0 
2 4 3
98
0 
1 8
30
0 
5 4 6 3 1 11
72
0 
9 2 12 1 9 3 10 5 8 6
74
0 
3 5 11 8
38
0 
3 3 4 5
9
0 
4 6 8 5 9
75
0 
10 2 11 8 4 1 6 12 9 3 7
9
0 
3 12 11 2
54
0 
0 
44
0 
8 7 1 6 5 3 10 8 9
15
0 
5 2 3 5 8 12
5
0 
7 3 12 9 7 10 6 5
18
0 
9 1 6 5 4 2 11 7 12 9
61
0 
7 1 6 2 5 12 7 10
57
0 
7 7 1 6 3 11 4 5
90
0 
4 12 3 4 1
92
0 
7 9 3 2 5 7 10 8
92
0 
2 9 5
81
0 
1 4
42
0 
10 5 7 12 3 10 9 8 2 11 4
98
0 
10 8 9 12 7 3 1 5 2 4 11
15
0 
4 3 11 2 6
63
0 
8 9 1 6 3 11 5 4 10
91
0 
9 5 8 3 11 7 6 2 1 12
83
0 
9 11 3 7 12 1 10 6 8 5
57
0 
4 6 8 4 1
56
0 
2 10 1
25
0 
6 2 4 9 3 1 11
2
0 
9 7 4 5 9 6 10 8 11 1
87
0 
3 10 7 2
54
0 
5 3 2 6 8 12
91
0 
9 11 7 6 12 3 2 8 1 5
11
0 
0 
100
0 
2 12 2
90
0 
1 12
12
0 
7 11 2 5 3 1 12 10
75
0 
3 12 8 11
91
0 
7 5 8 4 1 12 7 11
74
0 
2 3 10
12
0 
3 5 1 4
96
0 
7 8 5 2 4 6 11 10
3
0 
10 2 10 3 11 1 5 6 8 9 4
87
0 
2 9 5
95
0 
6 6 7 12 5 11 3
22
0 
10 5 3 1 2 9 6 4 10 8 7
83
0 
2 10 3
66
0 
0 
67
0 
6 6 12 8 3 11 4
50
0 
9 5 2 10 4 8 6 9 1 11
79
0 
7 9 12 8 4 6 10 2
17
0 
2 12 4
58
0 
4 4 9 2 10
17
0 
2 6 4
51
0 
7 9 4 11 8 12 10 1
52
0 
3 1 5 4
85
0 
3 11 6 8
48
0 
0 
14
0 
9 4 5 7 3 10 9 6 2 1
36
0 
5 1 7 11 2 10
84
0 
2 1 7
85
0 
4 2 10 3 5
60
0 
9 5 12 8 4 10 11 2 7 6
5
0 
9 10 1 2 12 9 4 3 7 6
67
0 
6 10 7 5 1 9 3
70
0 
8 2 12 4 1 6 5 11 3
51
0 
2 8 5
87
0 
9 7 4 8 12 5 6 3 9 1
88
0 
5 1 2 8 6 10
25
0 
3 1 7 10
36
0 
2 12 8
13
0 
4 9 3 1 6
33
0 
4 12 2 4 5
40
0 
5 8 1 6 9 2
25
0 
3 9 6 12
81
0 
1 11
78
0 
7 4 11 12 3 7 5 8
72
0 
2 9 10
54
0 
3 10 4 3
62
0 
4 6 5 2 3
70
0 
10 3 1 11 7 5 4 9 10 2 8
9
0 
8 5 12 4 7 3 9 11 2
18
0 
3 9 5 3
52
0 
5 7 4 2 8 10
55
0 
6 2 7 8 4 5 12
18
0 
3 12 1 7
12
0 
7 4 2 7 12 9 5 8
98
0 
6 10 8 5 4 7 9
4
0 
6 1 8 11 2 9 5
84
0 
5 3 4 11 8 2
31
0 
0 
67
0 
10 1 5 7 2 6 8 11 12 4 9
99
0 
4 9 10 11 3
11
0 
8 10 11 6 3 9 8 12 2
47
0 
7 10 12 7 1 4 11 3
65
0 
8 2 8 9 11 12 5 6 4
9
0 
10 10 3 8 2 5 4 6 11 1 9
90
0 
7 8 1 9 4 12 10 3
97
0 
10 5 4 10 1 9 3 8 2 11 6
9
0 
5 7 1 8 3 12
54
0 
0 
79
0 
0 
34
0 
7 1 11 2 12 7 6 4
2
0 
0 
13
0 
1 4
84
0 
10 11 8 12 4 5 1 6 2 7 10
25
0 
1 11
29
0 
2 1 7
62
0 
3 2 12 1
36
0 
2 2 10
46
0 
2 11 4
63
0 
2 2 1
16
0 
0 
79
0 
10 11 2 5 6 3 10 4 7 8 12
3
0 
10 12 11 9 1 7 3 2 4 6 8
51
0 
4 1 8 5 6
88
0 
3 10 4 9
64
0 
10 7 11 10 4 8 12 6 5 2 1
78
0 
7 7 10 8 5 2 3 1
90
0 
5 10 11 3 4 1
75
0 
5 11 4 1 6 5
92
0 
9 6 7 8 9 10 1 3 2 12
2
0 
7 12 5 9 7 4 3 2
57
0 
6 7 1 12 9 5 3
99
0 
0 
95
0 
9 6 8 7 3 12 2 11 4 10
92
0 
2 2 1
0
0 
7 2 12 10 8 3 4 5
7
0 
8 7 11 1 9 8 10 2 4
30
0 
9 10 2 6 1 11 5 12 8 7
59
0 
6 2 5 1 10 12 6
100
0 
6 8 1 9 3 6 12
72
0 
3 11 10 7
11
0 
10 4 5 2 11 12 6 7 9 1 8
60
0 
1 12
12
0 
8 7 5 4 1 11 8 10 3
27
0 
7 2 4 5 7 6 10 12
81
0 
4 1 6 3 7
19
0 
6 11 2 4 5 12 1
78
0 
2 3 8
46
0 
6 10 4 1 2 9 3
19
0 
1 9
97
0 
1 3
53
0 
5 6 3 11 7 5
31
0 
9 6 3 8 12 5 4 1 2 11
23
0 
6 9 8 1 7 4 6
95
0 
0 
83
0 
8 8 10 1 12 6 5 7 3
18
0 
5 5 9 10 8 1
100
0 
2 5 1
83
0 
8 3 12 8 6 1 9 5 7
0
0 
7 12 7 9 2 4 1 3
38
0 
2 6 12
50
0 
5 6 3 1 7 9
70
0 
4 7 12 5 6
64
0 
6 12 9 10 6 8 2
23
0 
5 12 8 6 10 5
26
0 
9 5 3 10 12 7 11 2 8 1
13
0 
2 3 1
46
0 
4 11 7 3 8
6
0 
10 7 3 11 6 2 12 8 4 5 9
11
0 
10 12 4 5 7 9 1 3 11 6 2
86
0 
2 5 1
11
0 
2 5 4
44
0 
1 12
88
0 
2 5 6
11
0 
7 2 7 1 8 11 12 9
16
0 
8 7 4 1 5 8 12 6 9
29
0 
8 8 11 5 4 2 12 1 9
89
0 
9 4 11 12 2 6 7 3 9 5
79
0 
6 5 11 7 2 3 12
59
0 
4 1 6 2 7
16
0 
0 
50
0 
8 5 7 9 1 6 12 11 3
14
0 
5 7 10 5 3 8
37
0 
4 12 9 1 4
41
0 
4 7 10 2 1
91
0 
1 5
79
0 
4 5 3 6 11
53
0 
1 3
46
0 
5 1 5 9 10 3
40
0 
8 5 2 4 3 7 1 9 12
87
0 
5 9 10 4 3 5
92
0 
2 11 12
11
0 
7 4 5 10 1 2 6 3
62
0 
1 4
25
0 
2 2 4
8
0 
6 11 2 6 9 10 5
34
0 
5 9 5 1 2 6
36
0 
5 2 9 3 7 1
10
0 
8 4 9 2 12 10 3 8 5
98
0 
9 9 5 6 10 2 1 4 12 3
23
0 
2 12 11
57
0 
7 2 8 12 11 4 9 6
59
0 
10 8 11 9 3 6 5 12 10 7 4
63
0 
3 6 5 11
35
0 
2 2 7
27
0 
6 11 12 8 4 2 10
3
0 
10 3 12 7 2 1 5 9 6 8 11
100
0 
9 1 11 4 8 10 5 3 7 12
83
0 
3 9 4 12
53
0 
5 4 1 6 7 12
96
0 
8 5 2 4 1 9 10 3 11
76
0 
9 5 6 9 1 11 7 10 2 3
47
0 
5 10 2 5 9 1
37
0 
1 8
46
0 
8 10 1 2 3 7 8 11 12
35
0 
9 1 12 6 5 9 7 10 11 2
78
0 
6 7 4 11 1 10 9
16
0 
9 12 3 6 4 2 5 10 11 8
11
0 
3 2 10 3
67
0 
5 3 7 10 8 1
96
0 
4 1 2 11 12
41
0 
9 1 8 12 6 4 11 10 7 3
44
0 
0 
20
0 
4 9 3 6 12
4
0 
8 2 8 4 11 1 6 10 9
67
0 
2 11 3
15
0 
8 11 5 10 7 8 1 3 4
18
0 
3 9 6 7
30
0 
3 12 7 3
92
0 
9 2 10 8 5 4 6 11 9 7
80
0 
7 2 1 9 10 12 8 3
100
0 
2 6 3
62
0 
7 7 12 8 1 4 2 6
13
0 
4 7 1 5 8
67
0 
2 2 1
44
0 
8 1 4 8 9 3 12 2 5
14
0 
8 9 1 10 7 5 8 12 6
58
0 
10 6 11 4 10 12 5 9 1 2 3
19
0 
9 7 4 10 12 1 11 3 5 2
73
0 
9 2 10 12 1 6 4 9 11 7
29
0 
6 8 5 4 1 7 9
89
0 
1 7
64
0 
3 7 6 10
87
0 
1 11
82
0 
10 3 8 10 11 12 6 1 2 7 5
34
0 
6 10 1 2 11 12 9
68
0 
9 8 7 6 10 3 2 1 12 11
24
0 
0 
27
0 
0 
0
0 
10 9 11 6 4 12 5 3 8 7 1
62
0 
3 4 11 1
86
0 
10 10 5 2 12 4 11 1 7 3 9
38
0 
7 10 1 6 5 8 3 11
81
0 
3 4 8 6
22
0 
2 2 3
47
0 
6 12 5 11 8 3 2
20
0 
6 7 2 11 1 4 3
48
0 
0 
21
0 
4 9 7 1 11
43
0 
2 11 6
9
0 
2 10 7
14
0 
5 3 11 2 12 5
22
0 
10 4 11 5 2 6 8 12 9 3 10
80
0 
9 1 10 2 9 6 11 12 4 7
51
0 
10 6 8 1 12 11 3 10 2 4 5
21
0 
0 
44
0 
4 7 10 12 6
32
0 
7 4 12 10 7 8 1 9
13
0 
9 8 9 7 3 5 1 10 12 6
98
0 
2 7 6
76
0 
3 10 4 12
93
0 
10 10 2 3 12 1 6 4 7 8 5
67
0 
9 9 3 4 10 12 1 2 11 6
72
0 
2 7 5
93
0 
3 6 5 8
92
0 
3 9 11 12
50
0 
5 10 3 2 11 5
69
0 
2 9 11
1
0 
0 
18
0 
0 
99
0 
0 
8
0 
6 11 9 5 7 2 1
48
0 
5 1 8 10 3 9
52
0 
3 1 11 12
63
0 
9 10 5 1 9 2 8 12 4 3
30
0 
7 4 12 11 9 10 3 8
87
0 
6 2 8 6 11 5 3
66
0 
2 5 2
26
0 
1 8
25
0 
3 2 8 3
56
0 
10 3 7 1 2 5 12 10 4 6 8
61
0 
6 4 7 5 10 2 6
47
