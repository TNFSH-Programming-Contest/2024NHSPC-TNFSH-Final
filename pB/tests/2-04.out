4
13651208
44
0
1656596
78242
30
15385843
1656534
19
245409
160
155029
90
77884
71
47
1566778
39
15385563
57
13663876
135
91
379
1643892
13896860
1489052
1734751
19
54
13728959
4
97
1722107
61
1566865
18
13741639
16
1488923
17
155045
12689
17
83
12643
155285
15372971
88
102
74
13014
7
204
90
78229
15
201
47
245600
245885
10
15140075
56
12981
13896860
1644368
79
88
77
131
13651210
11
75
1579706
233186
58
232987
103
12
1656978
2
50
1566813
27
36
12684
4
13896960
13896722
15
1643904
20
14
78
40
15385864
49
19
